app/components/earthworks/code_snippet_modal_component.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= render Blacklight::System::ModalComponent.new do |component| %>
+  <% component.with_title { t('geoblacklight.code_snippet.title') } %>
```

Comment:
`geoblacklight.code_snippet.title` is not coming through. Additionally, I would think this should go in the earthworks section, not geoblacklight. If you add it to `en.yml` under `earthworks`, and substitute the string location, it works for me locally:

`  <% component.with_title { t('earthworks.code_snippet.title') } %>`

Reply:
I used your suggested approach and it worked fine. Thanks!